app/assets/javascripts/uploadcare/settings.coffee

```diff
@@ -201,7 +203,18 @@ uploadcare.namespace 'settings', (ns) ->
 
     if settings.validators
       settings.validators = settings.validators.slice()
-
+    
+    if settings.previewProxy and not settings.previewUrlCallback
+      settings.previewUrlCallback = (url, info) =>
+        encodedUrl = encodeURIComponent(url)
+        justAppend = /\=$/.test(settings.previewProxy)
+        useAmpersand = /[^\&\?]$/.test(settings.previewProxy)
+
+        path = if justAppend then encodedUrl else
```

Comment:
`path` is part of URL after domain and before querystring. This is not a part of a path.